test/evergreen.yml

```diff
@@ -2224,7 +2225,6 @@ tasks:
       - func: "compile wiredtiger docs"
 
   - name: doc-update
-    patchable: false
```

Comment:
That was removed on purpose so people don't upload the docs by mistake when running patches.

Reply:
This is only for testing, once it passes I'll revert it.

Reply:
Resolved